### DISCUSSION TITLE
🚀 FR - Allow multiple {paginate} tags, displaying in different locations

### DISCUSSION THREAD

**Discussion by agun** [ExpressionEngine/ExpressionEngine#3835] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**
It would be really nice to be able to have 2 different `{paginate}` tags in a channel entries/search results/etc tag. 

A use case for this would be for being able to set one at the top which would say something like - _Showing results page 3 of 7_ … and one at the bottom which would then be the links to the new pages. 

Currently you can only have 1 pagination tag, that is displayed at either the top/bottom or both. 

**Describe the solution you'd like**
Moving the `paginate="xxx"` attribute away from the channel entries/search results/etc tag, and adding it to the {paginate} tag would make sense (maybe rename it as {paginate:bottom} or {paginate:top} or {paginate:inline} etc etc?). 

An example might be something like:

```
{exp:channel:entries channel="news" limit="10"}

  <div><a href="/news/{url_title}">{title}</a></div>

  {paginate:top}
    <div>Showing {current_page} of {total_pages}</div>
  {/paginate:top}

  {paginate:bottom}
    {pagination_links}
  {/paginate:bottom}

{/exp:channel:entries}

```

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by intoeetive** [comment]
We've started work on this: https://github.com/ExpressionEngine/ExpressionEngine/pull/3896
